docs/src/main/sphinx/connector/jdbc-batch-size.fragment

```diff
@@ -0,0 +1,20 @@
+Manage write batch size
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I don't think the word "manage" adds any context to the title. What about "Set the batch size" or "Manually set the batch size"